template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CmTemplateProcessor.java

```diff
@@ -78,6 +78,10 @@
 
     private static final String ROLE_TYPE = "roleType";
 
+    private static final BigDecimal SECURE_DATANODE_ADDRESS = BigDecimal.valueOf(9866L);
```

Comment:
should we add these ports to the deploymentconfig?

Reply:
What do you mean?

Reply:
Done